### DISCUSSION TITLE
Request Widget bottomNavigationBar

### DISCUSSION THREAD

**Discussion by bobwatcherx** [flet-dev/flet#577] [Q&A]
i want to make a bottomnavigatiobar, is there any other way around it if i create a container and i put it in the bottom corner, but it doesn't work
![image](https://user-images.githubusercontent.com/52072811/202130713-1a0bcd3f-959c-4d20-9ba8-4514f1e6a4ae.png)

i code my flet but not work
this my code 
```
import flet as ft
from flet import *
from flet import colors,icons
def main(page: ft.Page):
    
    page.add(
        Column([
         Container(
            bgcolor="blue500",
                alignment=alignment.center,
                content=Row([
                    IconButton(icon=icons.HOME),
                    IconButton(icon=icons.HOME),
                    IconButton(icon=icons.HOME),
                    IconButton(icon=icons.HOME),

                    ],alignment="spaceBetween")
                )
            ],alignment="end")
        )

ft.app(target=main, view=ft.WEB_BROWSER)
```

**and is there an Align widget like in flutter to set the button fixed to the middle**
thnks

**Comment by ndonkoHenri** [comment]
https://flet.dev/docs/controls/navigationbar
